sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -612,7 +612,294 @@ VALUES ('sdoh_instruments', 'hunger_vital_sign', 'Hunger Vital Sign (2-item)', 1
+       ('tribal_affiliations','cherokee_nation','Cherokee Nation (OK)',20,'40',1),
+       ('tribal_affiliations','chickasaw_nation','Chickasaw Nation (OK)',30,'43',1),
+       ('tribal_affiliations','choctaw_nation','Choctaw Nation of Oklahoma',40,'47',1),
+       ('tribal_affiliations','gila_river','Gila River Indian Community (AZ)',50,'93',1),
+       ('tribal_affiliations','hopi','Hopi Tribe (AZ)',60,'104',1),
+       ('tribal_affiliations','navajo_nation','Navajo Nation (AZ/NM/UT)',70,'170',1),
+       ('tribal_affiliations','standing_rock','Standing Rock Sioux Tribe (ND/SD)',80,'289',1),
+       ('tribal_affiliations','tohono_oodham','Tohono O''odham Nation (AZ)',90,'302',1),
+       ('tribal_affiliations','white_mountain_apache','White Mountain Apache Tribe (AZ)',100,'325',1),
+       ('tribal_affiliations','zuni','Zuni Tribe (NM)',110,'337',1),
+       ('tribal_affiliations','other_specify','Other (specify)',120,'000',1)
+ON DUPLICATE KEY UPDATE title=VALUES(title), notes=VALUES(notes), activity=VALUES(activity);
+#EndIf
+
+/* -------- ODH Industry (optional; used if you capture industry) -------- */
+#IfNotRow2D list_options list_id Industry option_id 541110
+INSERT INTO list_options (list_id, option_id, title, seq, is_default, option_value, notes, activity)
+VALUES ('lists','Industry','ODH Industry',0,0,0,'NAICS-based industry codes from ODH',1)
+ON DUPLICATE KEY UPDATE title=VALUES(title), notes=VALUES(notes);
+
+INSERT INTO list_options (list_id, option_id, title, seq, codes, activity)
+VALUES ('Industry','541110','Offices of Lawyers',10,'541110.008099',1),
+       ('Industry','541330','Engineering Services',20,'541330.008117',1),
+       ('Industry','236220','Commercial and Institutional Building Construction',30,'236220.004781',1),
+       ('Industry','622110','General Medical and Surgical Hospitals',40,'622110.009243',1),
+       ('Industry','611110','Elementary and Secondary Schools',50,'611110.008684',1),
+       ('Industry','561720','Janitorial Services',60,'561720.002294',1),
+       ('Industry','722511','Full-Service Restaurants',70,'722511.010339',1),
+       ('Industry','445110','Supermarkets and Other Grocery Stores',80,'445110.006564',1),
+       ('Industry','238210','Electrical Contractors',90,'238210.004871',1),
+       ('Industry','621111','Offices of Physicians (except Mental Health)',100,'621111.009165',1),
+       ('Industry','531110','Lessors of Residential Buildings',110,'531110.007615',1),
+       ('Industry','484121','General Freight Trucking, Long-Distance',120,'484121.007193',1),
+       ('Industry','812111','Barber Shops',130,'812111.011099',1),
+       ('Industry','522110','Commercial Banking',140,'522110.007773',1),
+       ('Industry','999999','Unemployed',150,'999999',1),
+       ('Industry','UNKNOWN','Unknown',160,'UNKNOWN',1)
+ON DUPLICATE KEY UPDATE option_id=VALUES(option_id), title=VALUES(title), codes=VALUES(codes), activity=VALUES(activity);
```

Comment:
hi @sjpadgett , 
Still don't understand:
1. Why using the `ON DUPLICATE KEY` in all these queries
2. And still seems like the original 3 Industry list items will remain in the upgrade database, but not be in the new database?

Reply:
The original 3 industry items aren't valid in any useful way so for new database we don't want them and going forward use only correct value sets.
Left the three legacy values in and appended the correct coded list. Did so to maintain legacy data. My preference is to delete the list and replace with new corrected list values.

I don't get what the big deal is. There are actually several more lists that need to go or be updated to reflect modern reporting. I try hard to impact users as little as possible but sometime it's necessary.
I removed the duplicate test on new lists but I feel more comfortable having the duplicate checks.